google-cloud-examples/src/main/java/com/google/cloud/examples/storage/objects/RotateObjectEncryptionKey.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.examples.storage.objects;
+
+// [START storage_rotate_encryption_key]
+import com.google.cloud.storage.BlobId;
+import com.google.cloud.storage.Storage;
+import com.google.cloud.storage.StorageOptions;
+
+public class RotateObjectEncryptionKey {
+  public static void rotateObjectEncryptionKey(
+      String projectId,
+      String bucketName,
+      String objectName,
+      String oldEncryptionKey,
+      String newEncryptionKey) {
+    // The ID of your GCP project
+    // String projectId = "your-project-id";
+
+    // The ID of your GCS bucket
+    // String bucketName = "your-unique-bucket-name";
+
+    // The ID of your GCS object
+    // String objectName = "your-object-name";
+
+    // The encryption key originally used to encrypt the object
+    // String oldEncryptionKey = "TIbv/fjexq+VmtXzAlc63J4z5kFmWJ6NdAPQulQBT7g="
```

Comment:
Explain that these are base64 encoded. Also could you add context of what kind of encryption key it needs to be?